acceptance/data/fixtures.rb

```diff
@@ -60,7 +62,8 @@ def stuffs_ddl_statement
               dates ARRAY<DATE>,
               timestamps ARRAY<TIMESTAMP>,
               numerics ARRAY<NUMERIC>,
-              json_array ARRAY<JSON>
+              json_array ARRAY<JSON>,
+              interval INTERVAL
```

Comment:
I notice the acceptance tests are failing because the backend doesn't like your DDL where you're creating a table with an interval-typed column. See https://btx.cloud.google.com/invocations/1f742ae9-a620-4d0b-afcc-e861cce9c1f6/targets/cloud-devrel%2Fclient-libraries%2Fruby-spanner%2Fpresubmit%2Facceptance/log. I suspect it's because you named the column `interval` which is now a type name and thus reserved. (I believe DDLs are case-insensitive.)